package.json

```diff
@@ -2,6 +2,7 @@
   "private": true,
   "scripts": {
     "lint": "eslint --ext .js,.mjs,.ts,.vue .",
+    "linkinator": "linkinator dist/ --recurse --skip 'twitter.com|www.farmlands.co.nz|www.instagram.com'",
```

Comment:
@MeganSteenkamp Change: `--skip '$https://'

Avoids devs needing to update this config each time a new external link is added

Reply:
@MeganSteenkamp what does recurse do? Is it possible that it is recursing on the farmlands page? Also would using a configuration file be useful? Looks like you can specify most options including `skip` https://www.npmjs.com/package/linkinator

Reply:
@tom-blake there are some external links that are working i.e. LinkedIn 🤔